R/loo.R

```diff
@@ -215,16 +218,18 @@ loo.matrix <-
            ...,
            r_eff = NULL,
            save_psis = FALSE,
-           cores = getOption("mc.cores", 1)) {
-
+           cores = getOption("mc.cores", 1),
+           is_method = c("psis", "tis", "sis")) {
     if (is.null(r_eff)) throw_loo_r_eff_warning()
-    psis_out <- psis.matrix(log_ratios = -x, r_eff = r_eff, cores = cores)
+    is_method <- match.arg(is_method)
+    psis_out <- importance_sampling.matrix(log_ratios = -x, r_eff = r_eff, cores = cores, method = is_method)
```

Comment:
So this is a more general question. I first started to call it is_object. But then that would have meant that I would have changed the API argument `save_psis`, and changing that would have been a large (even not back compatible) change in the API. So this can of course be done, but the question is then what we should do with the save_psis argument. Now I did as little as possible changes.

Reply:
yeah, I get it. Let's keep it `save_psis` for now, which we may as well eventually deprecate in favor of another name. But I agree this would be too much for now.